### DISCUSSION TITLE
Where is Location field set?

### DISCUSSION THREAD

**Discussion by rkelleyrtp** [lxdware/lxd-dashboard#29] [Q&A]
Can you please tell me what value is used for the Location field in the container listing?  I thought it was set to the LXD server (name or IP), but this does not appear to be true.  We have hundreds of containers spread across +20 LXD Servers and wold like to use the the location field to track down where a container is running.

Note: we are not using clustering - just standalone LXD Servers.

**Comment by matthewalanpenning** [comment]
Great question. The location field is used to indicate which LXD server an instance is actually located on when it is within a cluster. So for non-clustered LXD servers that value is set to "none". In clustered environments, you can select which host within the cluster to create a new instance, or if not selected LXD will automatically assign which host a new instance is created on.
